### PR TITLE
Konradstaniec/testnet parameters

### PR DESCRIPTION
**Pull request by KonradStaniec**
- Upgrades parameters in upgrade for testnet launch

### CODE REVIEW

**Review by Lazar955 — APPROVED** [review]

**Review by gitferry — APPROVED** [review]
lgtm!

File: app/upgrades/v1/testnet/btcstaking_params.go [hunk 1]
```diff
@@ -144,5 +144,32 @@ const BtcStakingParamsStr = `[
...
+    "max_staking_time_blocks": 64000,
+    "slashing_pk_script": "ABRb4SYk0IorQkCV18ByIcM0UNFL8Q==",
+    "min_slashing_tx_fee_sat": 5000,
+    "slashing_rate": "0.05",
+    "unbonding_time_blocks": 1008,
+    "unbonding_fee_sat": 2000,
+    "min_commission_rate": "0.03",
+    "delegation_creation_base_gas_fee": 1095000,
+    "allow_list_expiration_height": 26124,
+    "btc_activation_height": 229724
```
Comment (filippos47) [inline app/upgrades/v1/testnet/btcstaking_params.go:173]: Why this height?

**Review by filippos47 — APPROVED** [review]
Verified from the doc, one question only.

File: app/upgrades/v1/testnet/btcstaking_params.go (hunk 1 above)
Comment (filippos47) [reply app/upgrades/v1/testnet/btcstaking_params.go:173]: Staking is closed on testnet so no new stakes can be created. And unbonding on the existing stakes will not be affected by subsequent param versions.

Comment (KonradStaniec) [reply app/upgrades/v1/testnet/btcstaking_params.go:173]: good point ! will add current tip

File: app/upgrades/v1/upgrades_test.go [hunk 2]
```diff
@@ -152,13 +156,44 @@ func (s *UpgradeTestSuite) TestUpgrade() {
 				s.NoError(err)
 				s.EqualValues(resp.CodeID, 1)
 				s.Equal(stakingWasmChecksum[:], wasmvmtypes.Checksum(resp.Checksum))
+
+				// check that the gov params were updated
+				govParams, err := s.app.GovKeeper.Params.Get(s.ctx)
```
Comment (RafilxTenfen) [inline app/upgrades/v1/upgrades_test.go:161]: 👍

File: app/upgrades/v1/testnet/btcstaking_params.go (hunk 1 above)
Comment (vitsalis) [reply app/upgrades/v1/testnet/btcstaking_params.go]: ack

**Review by Lazar955 — APPROVED** [review]

**Pull request merged**